graphite/graphite.go

```diff
@@ -79,6 +82,20 @@ func (f *GraphitePublisher) Publish(metrics []plugin.Metric, cfg plugin.Config)
 		}
 		data := fmt.Sprintf("%v", m.Data)
 		logger.Debug("Metric ready to send %s:%s", key, data)
+
+		r := strings.NewReplacer(" ", "_",
```

Comment:
can you move this definition out to where var illegal = "(), /{}" is and group them in this fashion:

var (
    illegal = "(), /{}"
    replacement = ....
)